app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,105 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* mixins, variables, etc. */
+
+$gray-medium-light: #eaeaea;
+
+/* universal */
+
+body {
+ padding-top: 60px;
+}
+
+section {
+ overflow: auto;
```

このタブが1だったり、2だったり4だったりするので、統一しましょう。
rubyに合わせて2に統一しましょうか

```suggestion
  overflow: auto;
```

---

タブ2に統一しました。